### PR TITLE
Enable Unity3D rules

### PR DESCRIPTION
**Pull request by funkydude**
Enable the (working) Unity3D rules

### CODE REVIEW

**Comment by J0WI** [comment]
Great work so far!

I also got mismatches on:
beta.unity3d.com
response.unity3d.com

WFM:
id.unity.com
analytics.cloud.unity3d.com
support.unity3d.com
kharma.unity3d.com

**Comment by funkydude** [comment]
@J0WI Damnit, it seems you accidentally revealed that unity.com is also a thing with its own subdomains!

File: src/chrome/content/rules/Unity.xml [hunk 1]
```diff
@@ -29,9 +32,20 @@
 	<target host="publisher.assetstore.unity3d.com" />
 	<target host="assetstore.unity3d.com" />
 	<target host="accounts.unity3d.com" />
+	<target host="support.unity3d.com" />
+	<target host="kharma.unity3d.com" />
+
+	<target host="id.unity.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Unity.xml]: + api.unity.com

**Comment by funkydude** [comment]
@J0WI Done

**Pull request merged**